src/rws_client.cpp

```diff
@@ -439,6 +456,24 @@ RWSClient::RWSResult RWSClient::setMotorsOff()
   return evaluatePOCOResult(httpPost(uri, content), evaluation_conditions);
 }
 
+RWSClient::RWSResult RWSClient::setSpeedRatio(unsigned int ratio)
+{
+  if(ratio > 100) throw std::out_of_range("Speed ratio argument out of range");
```

Comment:
Seeing as the message is for humans:

```suggestion
  if(ratio > 100) throw std::out_of_range("Speed ratio argument out of range (should be 0 <= ratio <= 100)");
```